red/df1.js

```diff
@@ -125,6 +125,16 @@ module.exports = function (RED) {
         //avoids warnings when we have a lot of DF1 In nodes
         this.setMaxListeners(0);
 
+        this.getDf1Session = async () => {

```

Comment:
If you are returning a Promise already, there's no need to include the `async` keyword on the function signature.